packages/graphql-hooks/src/createRefetchMutationsMap.js

```diff
@@ -19,9 +19,14 @@ export default function createRefetchMutationsMap(refetchAfterMutations) {
     if (paramType === 'string') {
       result[mutationInfo] = {}
     } else if (paramType === 'object') {
-      const { filter, mutation } = mutationInfo
+      const { filter, mutation, refetchOnMutationError } = mutationInfo
 
-      result[mutation] = { filter }
+      result[mutation] = {
+        filter,
+        refetchOnMutationError:
+          typeof refetchOnMutationError === 'undefined' ||
+          refetchOnMutationError
```

Comment:
why this logic? can't we assume the user will provide a sensible value? if this is to apply the default value, let's just default it to true when destructuring it above

Reply:
you're right, a default fallback is enough.